qt/scientific_interfaces/ISISReflectometry/test/Batch/RowProcessingAlgorithmTest.h

```diff
@@ -29,177 +36,178 @@ class RowProcessingAlgorithmTest : public CxxTest::TestSuite {
 
   void testExperimentSettings() {
     auto model = Batch(m_experiment, m_instrument, m_runsTable, m_slicing);
-    auto result = createAlgorithmRuntimeProps(model);
-    TS_ASSERT_EQUALS(result->getPropertyValue("AnalysisMode"), "MultiDetectorAnalysis");
-    TS_ASSERT_EQUALS(result->getPropertyValue("ReductionType"), "NonFlatSample");
-    TS_ASSERT_EQUALS(result->getPropertyValue("SummationType"), "SumInQ");
-    TS_ASSERT_EQUALS(result->getPropertyValue("IncludePartialBins"), "1");
-    TS_ASSERT_EQUALS(result->getPropertyValue("Debug"), "1");
-    TS_ASSERT_EQUALS(result->getPropertyValue("SubtractBackground"), "1");
-    TS_ASSERT_EQUALS(result->getPropertyValue("BackgroundCalculationMethod"), "Polynomial");
-    TS_ASSERT_EQUALS(result->getPropertyValue("DegreeOfPolynomial"), "3");
-    TS_ASSERT_EQUALS(result->getPropertyValue("CostFunction"), "Unweighted least squares");
-    TS_ASSERT_EQUALS(result->getPropertyValue("PolarizationAnalysis"), "1");
-    TS_ASSERT_EQUALS(result->getPropertyValue("FloodCorrection"), "Workspace");
-    TS_ASSERT_EQUALS(result->getPropertyValue("FloodWorkspace"), "test_workspace");
-    TS_ASSERT_EQUALS(result->getPropertyValue("StartOverlap"), "7.500000");
-    TS_ASSERT_EQUALS(result->getPropertyValue("EndOverlap"), "9.200000");
-    TS_ASSERT_EQUALS(result->getPropertyValue("Params"), "-0.02");
-    TS_ASSERT_EQUALS(result->getPropertyValue("ScaleRHSWorkspace"), "1");
+    auto result = RowProcessing::createAlgorithmRuntimeProps(model);
+    checkExperimentSettings(*result);
```

Comment:
Will some asserts be added here once the work is complete? I know there are some placeholders for for future tests below.